### ISSUE TITLE
Hyperparameters

### ISSUE THREAD

**Issue by hhxxttxsh** [rasoolfa/videocap#1]
In the main model training script: train_SeqToSeq_MeMLocSoft_R2.lua, have you already given all the optimal hyper-parameters in the list of cmd:option? For instance, do I need to tune the learning rate? With the current setting, it is hard to reproduce the exact results from your paper.
